testing/bazel/APKBUILD

```diff
@@ -0,0 +1,70 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=bazel
+pkgver=0.25.2
+pkgrel=0
+pkgdesc="A fast, scalable, multi-language and extensible build system"
+url="https://bazel.build"
+arch="all"
+license="Apache-2.0"
+options="!check" # No test suite from upstream
+depends="zip unzip"
+# openjdk8 is a must
+makedepends="bash openjdk8 python3"
+subpackages="
+	$pkgname-bash-completion:bash:noarch
+	$pkgname-zsh-completion:zsh:noarch
+	"
+source="https://github.com/bazelbuild/bazel/releases/download/0.25.2/bazel-$pkgver-dist.zip"
+builddir="$srcdir/"
+
+build() {
+#	cd "$builddir"
```

Comment:
this can be removed, yes ?

Reply:
Not yet, please. Let the build successfully finish. The package zip file unpacks in content right into src/. I do not know if it will cause any artifacts.

Reply:
Fixed. Thank you!